x-pack/plugins/security_solution/public/flyout/document_details/right/components/description.tsx

```diff
@@ -119,7 +120,17 @@ export const Description: FC = () => {
         </EuiTitle>
       </EuiFlexItem>
       <EuiFlexItem data-test-subj={DESCRIPTION_DETAILS_TEST_ID}>
-        {isAlert ? alertRuleDescription : '-'}
+        <p
+          css={css`
+            word-break: break-word;
```

Comment:
styling looks a lot like <LineClamp> [here](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/line_clamp/index.tsx#L31). consider reusing that component?

Reply:
aren't we trying to get away from styled components in favor of using emotion?

Reply:
ok I just tried to use the component  (which is not exported at the moment btw) and it's actually changing the behavior slightly because on top of cutting off text with `-webkit-line-clamp`, it's also managing `height` and `max-height`.
The UI is then
![Screenshot 2023-10-19 at 1 49 49 PM](https://github.com/elastic/kibana/assets/17276605/691aa3d8-280a-4d25-8500-1d58e2b11b2c)
instead of
![Screenshot 2023-10-19 at 1 49 45 PM](https://github.com/elastic/kibana/assets/17276605/425f44c6-f454-4092-a81e-178636c6f668)
which is not what UIUX wants...

More interestingly, the `LineClamp` component is actually used in a [single place](https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/public/common/components/event_details/investigation_guide_view.tsx#L70) at the moment, which funny enough will be nuked when we remove the old flyout (so very soon...).

Would you be ok with NOT using that component after all?